aks/examples/cnpack/README.md

```diff
@@ -76,14 +76,14 @@ If it is needed to get the value of `log_analytics_workspace_primary_shared_key`
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
 | <a name="input_admin_group_object_ids"></a> [admin\_group\_object\_ids](#input\_admin\_group\_object\_ids) | (Required) A list of Object IDs (GUIDs) of Azure Active Directory Groups which should have Owner Role on the Cluster. <br>  This is not the email address of the group, the GUID can be found in the Azure panel by searching for the AD Group<br>  NOTE: You will need Azure "Owner" role (not "Contributor") to attach an AD role to the Kubernetes cluster. | `list(any)` | n/a | yes |
-| <a name="input_az_monitor-user-managed-id"></a> [az\_monitor-user-managed-id](#input\_az\_monitor-user-managed-id) | The user managed identity to *create* for use with the Azure monitor-- at this time this does not accept existing user or system managed identity | `string` | `"tf-holoscan-identity"` | no |
+| <a name="input_az_monitor_user_managed_id"></a> [az\_monitor-user-managed-id](#input\_az\_monitor-user-managed-id) | The user managed identity to *create* for use with the Azure monitor-- at this time this does not accept existing user or system managed identity | `string` | `"tf-holoscan-identity"` | no |
```

Comment:
Needs text update `az\_monitor-user-managed-id`

Reply:
Thanks for catching. Fixed in #11